auditbeat/docs/fields.asciidoc

```diff
@@ -55,13 +55,21 @@ type: keyword
 The target path for symlinks.
 
 [float]
+=== `audit.file.source`
```

Comment:
When I read the name I assume it contains the filepath. When reading the text I see that source is used here differently. My wrong assumption mainly comes from filebeat as we use it there differently. I wonder if perhaps `audit.file.trigger` could be an alternative?

Reply:
I removed the `source` field (bad name + it wasn't useful for anything).